app/controllers/cart_staches_controller.rb

```diff
@@ -10,6 +11,7 @@ def destroy
     stache = Stache.find(params[:id])
     @cart.delete_stache(stache.id)
     session[:cart] = @cart.contents
-    redirect_to cart_path
+    flash.now[:removed] = "Successfully removed #{view_context.link_to stache.name, stache_path(stache)} from your cart.".html_safe
+    render 'carts/show'# redirect_to cart_path
```

Comment:
Since they are already on the cart_path when they click the remove from cart button, it's not really a redirect, it's just re-rendering the page they are already on. What are your thoughts on why it should be a redirect?